### ISSUE TITLE
`Lint/Syntax` unexpected token tIdentifier in generator template file

### ISSUE THREAD

**Issue by cickes** [rubocop/rubocop#11639]
Trying to create a custom template for generating model specs.  
`lib/templates/rspec/model/model_spec.rb `

This line is triggering a `Lint/Syntax` error which cannot be disabled, by design.
`let(:<%= singular_name %>) { create :<%= singular_name %> }`

Complete error message:  
```
E: Lint/Syntax: unexpected token tIDENTIFIER
(Using Ruby 3.2 parser; configure using TargetRubyVersion parameter, under AllCops)
  let(:<%= singular_name %>) { create :<%= singular_name %> }
```

Expected behavior: nothing triggered.  

## Rubocop Version 
```
1.46.0 (using Parser 3.2.1.0, rubocop-ast 1.26.0, running on ruby 3.2.1) [x86_64-linux]
  - rubocop-rails 2.17.4
```

**Comment by dvandersluis** [comment]
Could you name your template `model_spec.erb` instead? That might stop RuboCop from processing it. 

Alternately, you could at it (or the lib/templates directory) to the `Exclude` list. https://docs.rubocop.org/rubocop/configuration.html#includingexcluding-files

**Comment by koic** [comment]
Yeah, It seems that ERB-like syntax is used instead of Ruby. @dvandersluis's suggestion will solve the issue.
